.github/workflow/on-push-publish-to-npm.yml

```diff
@@ -0,0 +1,38 @@
+name: on-push-publish-to-npm
+on:
+  workflow_dispatch:
```

Comment:
I was thinking about https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release trigger. The release process may look like following:
- Bump package's versions somehow. manually or using a tool.
- Make a new release on github when ready
- Release is automatically published to npm

Reply:
Now it:
- should be triggered manually (as we are at an early stage I would prefer to avoid automatic packages publication)
- may be executed only on `main` branch
- request user input for what version should be changed
- fix all code formatting issues
- check code style, test and build packages to ensure they are valid
- change the version and publish it as a tag and to npm publish (after verification I'll remove `--no-git --no-publish`)

As soon as we ensure we have a stable process we can switch to automated packages publication on pushes to the `main` branch.